cdap-docs/cdap-apps/build.sh

```diff
@@ -135,7 +130,6 @@ function download_includes() {
   
   download_md_doc_file $base_target $hydrator_source database-plugins Database-batchsink.md
   download_md_doc_file $base_target $hydrator_source database-plugins Database-batchsource.md
-  download_md_doc_file $base_target $hydrator_source database-plugins Teradata-batchsource.md
```

Comment:
This was probably removed. The other lines that you're removing in this file were not removed, but moved into another module.